jenkins-scripts/dsl/_configs_/OSRFUNIXBase.groovy

```diff
@@ -38,6 +38,62 @@ class OSRFUNIXBase extends OSRFBase
              git clone https://github.com/gazebo-tooling/release-tools scripts -b \$RTOOLS_BRANCH
              """.stripIndent())
       }
+          publishers {
+            postBuildScripts {
+              steps{
+              conditionalSteps {
+                  condition {
+                      expression('(.)*gpu-nvidia(.)*','${NODE_LABELS}')
```

Comment:
Can not find the official documentation for the `NODE_LABELS` variable. If it works as expected, it is probably a white space separated list of labels. If that is the case, we can refine the regexp to match exactly `gpu-nvidia` and not other hypothetical variants line `no-gpu-nvidia`.

Reply:
The NODE_LABELS variable is documented in https://github.com/jenkinsci/job-dsl-plugin/wiki/User-Power-Moves as an option. And yes it's a white space separated list of variables since I think  Jenkins treats variables as that and matches against it. I will tune the regex so it matches only gpu-nvidia.